pkg/agent/openflow/pipeline.go

```diff
@@ -2086,6 +2131,89 @@ func (c *client) serviceEndpointGroup(groupID binding.GroupIDType, withSessionAf
 	return group
 }
 
+// serviceSNATFlow generates the flow that handles the Service traffic which is from gateway.
```

Comment:
Will it be used by ClusterIP traffic from gateway as well? If not, can we replace "service" with "nodeport"?

Reply:
In the future, it will be also used by ClusterIP.